Automattic/woocommerce-payments
# Remove "Estimated" option from the deposit list advanced filters

Fixes #7630

#### Changes proposed in this Pull Request

This PR removes the `Estimated` option from the advanced filters on the deposits list.

Along with the UI change, the removal of the `estimated` value from the `DepositStatus` TS type was needed. This results in many code changes unrelated to the focus of this PR.

**Before/After**

<img width="375" alt="image" src="https://github.com/Automattic/woocommerce-payments/assets/3147296/f861aea4-4789-4dfc-bf76-39f6ef1f0e83">
<img width="375" alt="image" src="https://github.com/Automattic/woocommerce-payments/assets/3147296/429c9019-be9d-4717-ba1f-dabf465b6b38">


**Notes**
* This PR doesn't remove the `Estimated` deposits from this list, handled in #7604.
* This PR doesn't remove the `Estimated` deposits from the exported CSV, handled in #7631.
* This PR breaks the **Payments → Overview → Next deposit** UI by deleting the `Estimated` text from the `DepositStatusChip`. This will be handled in a new issue replacing this UI TBC.
	<img width="700" alt="image" src="https://github.com/Automattic/woocommerce-payments/assets/3147296/6a89029f-73f2-4689-b015-0d3a63a4fcc3">

#### Testing instructions





* Visit the **Payments → Deposits** screen.
* Select **Advanced Filters** at the top of the table.
* **Add a Filter** and select **Status**.
* Ensure `Estimated` is not an available option.
* Ensure filters work as expected.

-------------------


**Post merge**
